src/model/modifier/AtomicBlockUtils.js

```diff
@@ -84,6 +86,48 @@ const AtomicBlockUtils = {
 
     return EditorState.push(editorState, newContent, 'insert-fragment');
   },
+
+  moveAtomicBlockBefore: function(
+    editorState: EditorState,
+    contentBlock: ContentBlock
+  ): EditorState {
+    const contentState = editorState.getCurrentContent();
+    const selectionState = editorState.getSelection();
+
+    const withMovedAtomicBlock = moveBlockBeforeInContentState(
+      contentState,
+      selectionState,
+      contentBlock
+    );
+
+    const newContent = withMovedAtomicBlock.merge({
+      selectionBefore: selectionState,
+      selectionAfter: withMovedAtomicBlock.getSelectionAfter().set('hasFocus', true),
```

Comment:
Is this necessary? I see that it's being done in the existing method, just wondering if it's needed there or here.